.github/workflows/update-loci.yaml

```diff
@@ -41,18 +44,38 @@ jobs:
           conda info
 
       - name: Check loci and update bed files (short)
-        #if: ${{ github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'}}
+        if: ${{ github.event_name == 'pull_request'}}
         run: snakemake --config stages="skip-refs"
 
+      # Enable this step to open an SSH session for debugging
+      # - name: SSH debug
+      #   uses: mxschmitt/action-tmate@v3
+
+      - name: Find new literature (intermediate)
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          conda init
+          source /home/runner/.bashrc
+          conda activate strchive
+          snakemake --config stages="new-refs"
+
+      - name: Update all literature (long)
+        if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'}}
```

Comment:
So the long and intermediate steps will both run on workflow dispatches? Just checking that this is what you intended.

Reply:
The reason I made this comment originally is because, because of how they're named "short intermediate long", I would expect them to be mutually exclusive, i.e. only one of them runs each workflow run. But I haven't looked inside the scripts to see what's actually going on yet, is that actually the case? It doesn't have to be; I'd just recommend removing the lengths from the names if not.

Looks like now there's a clear split/association between the three trigger types (pr, schedule, manual) and the three script lengths (short, intermediate, long). Note that schedule and manual will also trigger PR on subsequent runs. Here's a breakdown of the scenarios (in terms of automatic workflow runs):

1. Someone commits to `main` → nothing happens
1. Someone manually creates a PR → first commit, nothing happens → someone commits again → `pull_request` triggers, "short" script runs
1. Someone manually runs `workflow_dispatch` → "intermediate" script runs, PR opened → someone commits again to opened PR → `pull_request` triggers, "short" script runs
1. `schedule` runs once a month → "long" script runs, PR opened → someone commits again to opened PR → `pull_request` triggers, "short" script runs

If that looks okay, then the PR LGTM

Reply:
Thanks for breaking that down!
Yes, I think that is the desired behavior. Let's see if that all works out in practice.